### PR TITLE
Wip 315 syncronise branch

### PR DESCRIPTION
**Pull request by LancelotMarti**
* Its quite a large PR but it's only to catch up on the develop branch since many changes have been added (such as datetime management that i needed)

* I only added a dataset with synthetic data on COVID and solve the conflicts

* Easier to look at the changes made in the 4 last commits

### CODE REVIEW

**Comment by PaulineMauryL** [comment]
## Coverage report


 <details><summary>Click to see where and how coverage changed</summary><table><thead>
  <tr><th>File</th><th>Statements</th><th>Missing</th><th>Coverage</th><th>Coverage<br>(new stmts)</th><th>Lines missing</th></tr>
</thead>
<tbody><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-58a9fc5a17540b2fe4c3e40437bafa37681f53d6daf18f5d14cef65f52b483ad">app.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-58a9fc5a17540b2fe4c3e40437bafa37681f53d6daf18f5d14cef65f52b483ad"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-58a9fc5a17540b2fe4c3e40437bafa37681f53d6daf18f5d14cef65f52b483ad"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-58a9fc5a17540b2fe4c3e40437bafa37681f53d6daf18f5d14cef65f52b483ad"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-58a9fc5a17540b2fe4c3e40437bafa37681f53d6daf18f5d14cef65f52b483ad"></a></td><td><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-58a9fc5a17540b2fe4c3e40437bafa37681f53d6daf18f5d14cef65f52b483adR83-R83">83</a></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465">constants.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4ef5cc30e18c28ff2745a2889f8b0ff899cca1f8bbf06fc33d0333dde1f48465"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-6f14a4ab67fef3f3365196fae801d5411a894f4ad3b6a93b1025f75e5aa22c76">mongodb_admin.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-6f14a4ab67fef3f3365196fae801d5411a894f4ad3b6a93b1025f75e5aa22c76"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-6f14a4ab67fef3f3365196fae801d5411a894f4ad3b6a93b1025f75e5aa22c76"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-6f14a4ab67fef3f3365196fae801d5411a894f4ad3b6a93b1025f75e5aa22c76"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-6f14a4ab67fef3f3365196fae801d5411a894f4ad3b6a93b1025f75e5aa22c76"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/admin_database</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9ad1ab94646a8457b254274dfc31566c705d655327cd01c7213d889b630707ab">factory.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9ad1ab94646a8457b254274dfc31566c705d655327cd01c7213d889b630707ab"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9ad1ab94646a8457b254274dfc31566c705d655327cd01c7213d889b630707ab"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9ad1ab94646a8457b254274dfc31566c705d655327cd01c7213d889b630707ab"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9ad1ab94646a8457b254274dfc31566c705d655327cd01c7213d889b630707ab"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d5fc635aab0f9a9d211f6f8414715978b9d4cc474418ab3bd6786fa0372db765">utils.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d5fc635aab0f9a9d211f6f8414715978b9d4cc474418ab3bd6786fa0372db765"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d5fc635aab0f9a9d211f6f8414715978b9d4cc474418ab3bd6786fa0372db765"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d5fc635aab0f9a9d211f6f8414715978b9d4cc474418ab3bd6786fa0372db765"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d5fc635aab0f9a9d211f6f8414715978b9d4cc474418ab3bd6786fa0372db765"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/dataset_store</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-cbb2cefffc5b3aef3b25e6cfdfc8082f6511a9924ecac2e653e147faa94e3618">lru_dataset_store.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-cbb2cefffc5b3aef3b25e6cfdfc8082f6511a9924ecac2e653e147faa94e3618"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-cbb2cefffc5b3aef3b25e6cfdfc8082f6511a9924ecac2e653e147faa94e3618"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-cbb2cefffc5b3aef3b25e6cfdfc8082f6511a9924ecac2e653e147faa94e3618"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-cbb2cefffc5b3aef3b25e6cfdfc8082f6511a9924ecac2e653e147faa94e3618"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/dp_queries/dp_libraries</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-b473422d8b3c6f3eadb3ae6ff414f4d7e119f696c18454d41a0358e4aa6778d3">diffprivlib.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-b473422d8b3c6f3eadb3ae6ff414f4d7e119f696c18454d41a0358e4aa6778d3"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-b473422d8b3c6f3eadb3ae6ff414f4d7e119f696c18454d41a0358e4aa6778d3"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-b473422d8b3c6f3eadb3ae6ff414f4d7e119f696c18454d41a0358e4aa6778d3"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-b473422d8b3c6f3eadb3ae6ff414f4d7e119f696c18454d41a0358e4aa6778d3"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-487e7b392c40a6433d936f39861be55a93a8805fd5a8b211fb11f83e477c90d1">factory.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-487e7b392c40a6433d936f39861be55a93a8805fd5a8b211fb11f83e477c90d1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-487e7b392c40a6433d936f39861be55a93a8805fd5a8b211fb11f83e477c90d1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-487e7b392c40a6433d936f39861be55a93a8805fd5a8b211fb11f83e477c90d1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-487e7b392c40a6433d936f39861be55a93a8805fd5a8b211fb11f83e477c90d1"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb">opendp.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-fa24a03009808c4b8ce9362d10c36fdcdaa1b1043361cfd8964094bee91c1ecb"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-c9e56484759e6feee28ffb415d9596a0c8f3b445caade6ead99bd8e6086d322b">smartnoise_sql.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-c9e56484759e6feee28ffb415d9596a0c8f3b445caade6ead99bd8e6086d322b"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-c9e56484759e6feee28ffb415d9596a0c8f3b445caade6ead99bd8e6086d322b"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-c9e56484759e6feee28ffb415d9596a0c8f3b445caade6ead99bd8e6086d322b"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-c9e56484759e6feee28ffb415d9596a0c8f3b445caade6ead99bd8e6086d322b"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-bf8e93ac1c3b582ad4082be55cec7b0ef4eae5379fa751ea9a5bf6ecc8cfbc76">utils.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-bf8e93ac1c3b582ad4082be55cec7b0ef4eae5379fa751ea9a5bf6ecc8cfbc76"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-bf8e93ac1c3b582ad4082be55cec7b0ef4eae5379fa751ea9a5bf6ecc8cfbc76"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-bf8e93ac1c3b582ad4082be55cec7b0ef4eae5379fa751ea9a5bf6ecc8cfbc76"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-bf8e93ac1c3b582ad4082be55cec7b0ef4eae5379fa751ea9a5bf6ecc8cfbc76"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/private_dataset</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d2eef20d1bb0a82bc926e765b27b66d544eab827a91b02da0bd578e658232770">factory.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d2eef20d1bb0a82bc926e765b27b66d544eab827a91b02da0bd578e658232770"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d2eef20d1bb0a82bc926e765b27b66d544eab827a91b02da0bd578e658232770"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d2eef20d1bb0a82bc926e765b27b66d544eab827a91b02da0bd578e658232770"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d2eef20d1bb0a82bc926e765b27b66d544eab827a91b02da0bd578e658232770"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-a4f2583ec2eebdc72cefd3b982bb7df9689ca596455459384b1b3a22188f4c73">path_dataset.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-a4f2583ec2eebdc72cefd3b982bb7df9689ca596455459384b1b3a22188f4c73"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-a4f2583ec2eebdc72cefd3b982bb7df9689ca596455459384b1b3a22188f4c73"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-a4f2583ec2eebdc72cefd3b982bb7df9689ca596455459384b1b3a22188f4c73"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-a4f2583ec2eebdc72cefd3b982bb7df9689ca596455459384b1b3a22188f4c73"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-1e54c25951230aa65b4646aca131ee2ea874c4082887fcc6f1f924df6bd49d00">private_dataset.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-1e54c25951230aa65b4646aca131ee2ea874c4082887fcc6f1f924df6bd49d00"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-1e54c25951230aa65b4646aca131ee2ea874c4082887fcc6f1f924df6bd49d00"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-1e54c25951230aa65b4646aca131ee2ea874c4082887fcc6f1f924df6bd49d00"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-1e54c25951230aa65b4646aca131ee2ea874c4082887fcc6f1f924df6bd49d00"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/routes</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-99c0780dd8ed58717831157f6d4d562bb50c6fae37b794494b5815a2fdc57d59">routes_admin.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-99c0780dd8ed58717831157f6d4d562bb50c6fae37b794494b5815a2fdc57d59"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-99c0780dd8ed58717831157f6d4d562bb50c6fae37b794494b5815a2fdc57d59"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-99c0780dd8ed58717831157f6d4d562bb50c6fae37b794494b5815a2fdc57d59"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-99c0780dd8ed58717831157f6d4d562bb50c6fae37b794494b5815a2fdc57d59"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d88094fcb2b2b1f1caa40c3745c00ef392219864f6fe3c291e0b5ef9db0d72d9">routes_dp.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d88094fcb2b2b1f1caa40c3745c00ef392219864f6fe3c291e0b5ef9db0d72d9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d88094fcb2b2b1f1caa40c3745c00ef392219864f6fe3c291e0b5ef9db0d72d9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d88094fcb2b2b1f1caa40c3745c00ef392219864f6fe3c291e0b5ef9db0d72d9"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-d88094fcb2b2b1f1caa40c3745c00ef392219864f6fe3c291e0b5ef9db0d72d9"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-77d817538e954144e17cfdda5e9bbab94f751e44dd460710f202da5c77782b49">utils.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-77d817538e954144e17cfdda5e9bbab94f751e44dd460710f202da5c77782b49"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-77d817538e954144e17cfdda5e9bbab94f751e44dd460710f202da5c77782b49"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-77d817538e954144e17cfdda5e9bbab94f751e44dd460710f202da5c77782b49"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-77d817538e954144e17cfdda5e9bbab94f751e44dd460710f202da5c77782b49"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/tests</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c">test_api.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7f7d18385b324497d5ebcf51c3e0c0172158683cd7f916a75a261815c51f617c"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2">test_api_diffprivlib.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2"></a></td><td><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-0ce03d082552382cfaeadeba39a2db7a04f14d000a7ee3ef20e205a0dc05a0f2R27-R30">27-30</a></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc">test_mongodb_admin.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8860acb398146823e4b75c9bac2b9fc191e2f220076e2a3c57b9e20ffc2634bc"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7e12850d27b3e53e690a3a59c2dc3bf18468e648a98f2554b2781f882dfffc86">test_mongodb_admin_cli.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7e12850d27b3e53e690a3a59c2dc3bf18468e648a98f2554b2781f882dfffc86"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7e12850d27b3e53e690a3a59c2dc3bf18468e648a98f2554b2781f882dfffc86"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7e12850d27b3e53e690a3a59c2dc3bf18468e648a98f2554b2781f882dfffc86"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-7e12850d27b3e53e690a3a59c2dc3bf18468e648a98f2554b2781f882dfffc86"></a></td><td></td><tr>
<td colspan="6">&nbsp;&nbsp;<b>server/lomas_server/utils</b></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8131c38c3f82c3eb67b661f11276f5a7f34072188447f0d96fd9ff816d25db60">collection_models.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8131c38c3f82c3eb67b661f11276f5a7f34072188447f0d96fd9ff816d25db60"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8131c38c3f82c3eb67b661f11276f5a7f34072188447f0d96fd9ff816d25db60"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8131c38c3f82c3eb67b661f11276f5a7f34072188447f0d96fd9ff816d25db60"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-8131c38c3f82c3eb67b661f11276f5a7f34072188447f0d96fd9ff816d25db60"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9de6302bd7c6e2ee14146d4116b01cc419bc0d69da9456fb525f4b5d8ca2caef">config.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9de6302bd7c6e2ee14146d4116b01cc419bc0d69da9456fb525f4b5d8ca2caef"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9de6302bd7c6e2ee14146d4116b01cc419bc0d69da9456fb525f4b5d8ca2caef"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9de6302bd7c6e2ee14146d4116b01cc419bc0d69da9456fb525f4b5d8ca2caef"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-9de6302bd7c6e2ee14146d4116b01cc419bc0d69da9456fb525f4b5d8ca2caef"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-2fa5046d7b0bbe61fe4bb20a9e7eb645fd04ada9f9f725be9bb7fdc2642dd212">query_examples.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-2fa5046d7b0bbe61fe4bb20a9e7eb645fd04ada9f9f725be9bb7fdc2642dd212"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-2fa5046d7b0bbe61fe4bb20a9e7eb645fd04ada9f9f725be9bb7fdc2642dd212"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-2fa5046d7b0bbe61fe4bb20a9e7eb645fd04ada9f9f725be9bb7fdc2642dd212"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-2fa5046d7b0bbe61fe4bb20a9e7eb645fd04ada9f9f725be9bb7fdc2642dd212"></a></td><td></td><tr>
<td>&nbsp;&nbsp;<a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-53933ca754b4bd2c61d5f22f7fe02ba8fc76501988926a1cc0afc4d174c70c26">query_models.py</a></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-53933ca754b4bd2c61d5f22f7fe02ba8fc76501988926a1cc0afc4d174c70c26"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-53933ca754b4bd2c61d5f22f7fe02ba8fc76501988926a1cc0afc4d174c70c26"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-53933ca754b4bd2c61d5f22f7fe02ba8fc76501988926a1cc0afc4d174c70c26"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-53933ca754b4bd2c61d5f22f7fe02ba8fc76501988926a1cc0afc4d174c70c26"></a></td><td></td></tbody>
<tfoot>
<tr>
<td><b>Project Total</b></td>
<td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td align="center"><a href="https://github.com/dscc-admin-ch/lomas/pull/160/files#diff-4b0bf2efa3367c0072ac2bf1e234e703dc46b47aaa4fe9d3b01737b1a15752b1"></a></td><td>&nbsp;</td>
</tr>
</tfoot>
</table>

_The report is truncated to 25 files out of 34. To see the full report, please visit the workflow summary page._



<sub>

This report was generated by [python-coverage-comment-action](https://github.com/py-cov-action/python-coverage-comment-action)

</sub>
</details>

**Comment by PaulineMauryL** [comment]
How do you synchronise ? You branched of feature/opendp et rebased from master ? or something else?

File: docs/source/server_managing_dataset.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
+Adding and referencing new dataset
```
Comment (PaulineMauryL) [inline docs/source/server_managing_dataset.rst:1]: That's great!! thanks

File: docs/source/server_managing_dataset.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+2. Remote dataset (HTTP)
+If the dataset is stored externally and can be directly downloaded online.
+
+3. Remote dataset (S3)
+If the dataset is stored externally on a s3 instance (minio, aws, etc.)
+
+Once the type of dataset is figured out, one can modify the file `dataset_collection.yaml` in the folder `server/data/collections`.
+
+Example with our fake dataset, the administrator need to add this information (local example).
```
Comment (PaulineMauryL) [inline docs/source/server_managing_dataset.rst:22]: needS

Comment (LancelotMarti) [reply docs/source/server_managing_dataset.rst:22]: oups thanks

**Comment by LancelotMarti** [comment]
> How do you synchronise ? You branched of feature/opendp et rebased from master ? or something else?


I merged develop in my current branch and solved the conflicts (especially due of the restructuring of certain files). A rebase would take me forever.

**Review by PaulineMauryL — APPROVED** [review]

**Pull request merged**